src/qml/MapCanvas.qml

```diff
@@ -26,6 +26,10 @@ Item {
   property alias isRendering: mapCanvasWrapper.isRendering
   property alias incrementalRendering: mapCanvasWrapper.incrementalRendering
 
+  property int mouseX
+  property int mouseY
+  property bool skipClicked
```

Comment:
Can you put these into a child QtObject item? That will make them private vars (currently they are public API, not sure that's intended)

Reply:
If I do so, how could I access the properties from qgismobileapp? Making those private seems to only allow for the MapCanvas qml file to access the properties.

Reply:
My bad, you are right.
In this case, what about putting all the code into MapCanvas and emitting a "singleClick" signal instead?
That would make this reusable for other apps.

Reply:
Sounds like a good idea, I'll push that approach.

Reply:
FYI, signals / slots are used by naming convention in QML
you can add a signal `singleClick` and a slot `onSingleClick` will automatically be connected.